src/wasm-s-parser.h

```diff
@@ -1470,6 +1469,12 @@ class SExpressionWasmBuilder {
       } else if (inner[0]->str() == IMPORT) {
         importModule = inner[1]->str();
         importBase = inner[2]->str();
+        auto im = make_unique<Import>();
+        im->kind = ExternalKind::Memory;
+        im->module = importModule;
+        im->base = importBase;
+        im->name = importModule; // + importBase;
```

Comment:
This was an extraneous comment that I accidentally left in in the previous PR; you don't need to copy it :) I don't think the name actually matters; maybe @kripken knows better and maybe even we should come up with a better name for this PR I think it's fine to leave as-is, just without the comment. (Bonus points if you remove the corresponding comment from the table part :) )